app.json

```diff
@@ -32,6 +32,7 @@
     ],
     "ios": {
       "bundleIdentifier": "com.Stanford.Daily.App",
+      "buildNumber": "2",
```

Comment:
@hesyifei do you know why this wasn't needed before?

Reply:
Apple allows different `version` with the same `buildNumber` (default `1`), but doesn't allow same `version` with the same `buildNumber`. So I have to change it because I uploaded two `version` `1.2` to TestFlight.